### PR TITLE
Add RBAC and pod examples

### PR DESCRIPTION
**Pull request by squall0gd**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: cluster/addons/iso-client/examples/iso-client.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: coreaffinity-isolator
+  namespace: kube-system
+spec:
```
Comment (PiotrProkop) [inline cluster/addons/iso-client/examples/iso-client.yaml:6]: there should be another spec filed under template field and you have to add labels e.g. :
```
apiVersion: extensions/v1beta1
kind: DaemonSet
metadata:
  name: coreaffinity-isolator
  namespace: kube-system
spec:
  template:
    metadata:
      labels:
        isolator: core-affinity
    spec:
      serviceAccountName: coreaffinity-isolator-sa
      containers:
      - command:
        - /coreaffinity-isolator
        - -v
        - "4"
        - -logtostderr
        image: localhost:5000/iso-client:latest
        name: iso-client
        resources:
          requests:
            cpu: 250m
      hostNetwork: true
```

File: cluster/addons/iso-client/examples/iso-client-rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  - nodes/status
+  verbs:
+  - update
+  - patch
+  - get
+  - list
+  - delete
+---
+apiVersion: v1
+kind: ServiceAccount
```
Comment (flyingcougar) [inline cluster/addons/iso-client/examples/iso-client-rbac.yaml:19]: @squall0gd ServiceAccount should be in the same namespace as DaemonSet

File: cluster/addons/iso-client/examples/iso-client.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  template:
+    serviceAccountName: coreaffinity-isolator-sa
+    containers:
+    - command:
+      - /coreaffinity-isolator
```
Comment (flyingcougar) [inline cluster/addons/iso-client/examples/iso-client.yaml]: Binary name is `iso-client`
s/coreaffinity-isolator/iso-client

**Review by flyingcougar — CHANGES_REQUESTED** [review]

**Comment by PiotrProkop** [comment]
LGTM

**Review by flyingcougar — APPROVED** [review]

**Pull request merged**